extensions/vscode-colorize-tests/test/colorize-fixtures/test.js

```diff
@@ -4,20 +4,20 @@
  *--------------------------------------------------------------------------------------------*/
 
 var gulp = require('gulp');
-var tsb = require('gulp-tsb');
+var tsb = require('../../../../build/lib/tsb');
```

Comment:
@jrieken This is just a fixture... is this needed? 😕

Reply:
Find and replace style... Tho, the file is interpreted by the compiler and you will get an editor squiggle otherwise